paimon-web-server/src/test/java/org/apache/paimon/web/server/controller/SysRoleControllerTest.java

```diff
@@ -134,13 +158,41 @@ public void testEditRole() throws Exception {
         assertEquals(r.getData().getRoleName(), newRoleName);
     }
 
+    @Test
+    @Order(4)
+    public void testGetRoleList() throws Exception {
+        String responseString =
+                mockMvc.perform(
+                                MockMvcRequestBuilders.get(rolePath + "/list")
+                                        .cookie(cookie)
+                                        .contentType(MediaType.APPLICATION_JSON_VALUE)
+                                        .accept(MediaType.APPLICATION_JSON_VALUE))
+                        .andExpect(MockMvcResultMatchers.status().isOk())
+                        .andDo(MockMvcResultHandlers.print())
+                        .andReturn()
+                        .getResponse()
+                        .getContentAsString();
+
+        PageR<?> r = ObjectMapperUtils.fromJSON(responseString, PageR.class);
+        assertNotNull(r);
+        assertTrue(
```

Comment:
Could you assert the actual data content?

Reply:
Thanks for your review, I have updated this PR

Reply:
> Could you assert the actual data content?

Thanks for your review, I have updated this PR